app/components/map/index.js

```diff
@@ -730,10 +793,11 @@ class Map extends Component {
           minZoomLevel={2}
           maxZoomLevel={18}
           initialRegion={region}
-          rotateEnabled={false}
-          onRegionChangeComplete={this.updateRegion}
-          onMapReady={this.onMapReady}
+          showsCompass
```

Comment:
Are we using the compass?

Reply:
This is the top right circle that will appear on the map when you rotate it and it helps you to know where the north is.